assets/large_language_models/rag/environments/rag/context/conda_dependencies.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - scikit-learn
 - pip:
     # RAG package
-  - azureml-rag[azure,cognitive_search,data_generation,pinecone,milvus,azure_cosmos_mongo_vcore]==0.2.34
+  - azureml-rag[azure,cognitive_search,data_generation,pinecone,milvus,azure_cosmos_mongo_vcore]==0.2.35
```

Comment:
azureml-rag 0.2.35 now supports azure_cosmos_nosql extension to support azure cosmos db for nosql index, could we add it here?